bundles/org.openhab.binding.onkyo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -188,5 +188,29 @@
 		<description>Position of the currently selected menu item</description>
 		<state readOnly="true" pattern="%s"></state>
 	</channel-type>
+	<channel-type id="audioIn" advanced="true">
+		<item-type>String</item-type>
+		<label>Audio Input Info</label>
+		<description>Audio Input Stream Information</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="audioOut" advanced="true">
+		<item-type>String</item-type>
+		<label>Audio Output Info</label>
+		<description>Audio Output Stream Information</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="videoIn" advanced="true">
+		<item-type>String</item-type>
+		<label>Video Input Info</label>
+		<description>Video Input Stream Information</description>
+		<state readOnly="true" pattern="%s"></state>
```

Comment:
Is there a reason we specify a pattern for some of the new channels but not all ?

Reply:
None at all...  Did not notice it... 
I wrote the code 5years ago... and needed to download the  onkyo app just for this info and remembered I did write this way back. time to dust if off